File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.py
class Solution:
    def minOperations(self, s: str) -> int:
        cnt = 0

        for i in range(len(s)):
            if i % 2:
                if s[i] == '0': cnt += 1
            else:
                if s[i] == '1': cnt += 1

        return min(cnt, len(s)-cnt)

File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.scala
object Solution {
    def minOperations(s: String): Int = {
        val cnt = s.zipWithIndex.count {
            case (c, i) => i % 2 == 0 && c == '1' || i % 2 == 1 && c == '0'
        }

        Math.min(cnt, s.length - cnt)
    }
}
